Epoch 1:   0%|                                                                                                        | 0/3 [00:02<?, ?it/s]
Traceback (most recent call last):
  File "msldm_voice2instrumental/train.py", line 264, in <module>
    main()
  File "msldm_voice2instrumental/train.py", line 261, in main
    trainer.train()
  File "msldm_voice2instrumental/train.py", line 205, in train
    for batch in pbar:
  File "/Users/joaoesteves/mli/SingwithML/venv/lib/python3.8/site-packages/tqdm/std.py", line 1181, in __iter__
    for obj in iterable:
  File "/Users/joaoesteves/mli/SingwithML/venv/lib/python3.8/site-packages/torch/utils/data/dataloader.py", line 630, in __next__
    data = self._next_data()
  File "/Users/joaoesteves/mli/SingwithML/venv/lib/python3.8/site-packages/torch/utils/data/dataloader.py", line 1344, in _next_data
    return self._process_data(data)
  File "/Users/joaoesteves/mli/SingwithML/venv/lib/python3.8/site-packages/torch/utils/data/dataloader.py", line 1370, in _process_data
    data.reraise()
  File "/Users/joaoesteves/mli/SingwithML/venv/lib/python3.8/site-packages/torch/_utils.py", line 706, in reraise
    raise exception
AttributeError: Caught AttributeError in DataLoader worker process 0.
Original Traceback (most recent call last):
  File "/Users/joaoesteves/mli/SingwithML/venv/lib/python3.8/site-packages/torch/utils/data/_utils/worker.py", line 309, in _worker_loop
    data = fetcher.fetch(index)  # type: ignore[possibly-undefined]
  File "/Users/joaoesteves/mli/SingwithML/venv/lib/python3.8/site-packages/torch/utils/data/_utils/fetch.py", line 52, in fetch
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/Users/joaoesteves/mli/SingwithML/venv/lib/python3.8/site-packages/torch/utils/data/_utils/fetch.py", line 52, in <listcomp>
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/Users/joaoesteves/mli/SingwithML/msldm_voice2instrumental/data/dataset.py", line 62, in __getitem__
    vocal = self.preprocessor.load_and_preprocess(vocal_path)
  File "/Users/joaoesteves/mli/SingwithML/msldm_voice2instrumental/data/preprocessing.py", line 28, in load_and_preprocess
    waveform = self.high_pass_filter(waveform, 80)
  File "/Users/joaoesteves/mli/SingwithML/msldm_voice2instrumental/data/preprocessing.py", line 37, in high_pass_filter
    sos = librosa.filters.butter(6, cutoff, btype='high', fs=self.sample_rate)
AttributeError: module 'librosa.filters' has no attribute 'butter'
